about.md

```diff
@@ -0,0 +1,145 @@
+    control-your-metadata-with-yaml: Control Your Metadata With YAML
+  flexibility-and-customization:
+    _title: Flexibility and Customization
+    themes: Themes...
+    plugins: Plugins...
+  getting-started: Getting Started
+nav-url: /about/
+nav: 2
+---
+
+Pico is a stupidly simple, blazing fast, flat file CMS.  For those of us who aren't an expert webmaster, what does that even mean?  In the most basic sense, it means that there is no administration backend or database to deal with. You simply create .md files in the content folder and those files become your pages.  There's *much* more to Pico than that though.
+
+![Pico's Default Theme]({{ site.github.url }}/style/images/docs/pico_about_default_theme.png)
+Pico's default theme.  It's a bit bare... but that's intentional!
+
+* Center the text below images.
+
+Let's get this out of the way: Pico is *not* a turn-key solution.  If your goal is to deploy a generic-looking website without getting your hands dirty, Pico is probably not for you.
+
+* Is this too discouraging?
+
+That being said, Pico is incredibly extendable and customizable.  With a very small amount of configuration, you'll find yourself with a very personalized experience.  There's also a growing community creating ready-to-go [themes and plugins](#flexibility-and-customization) for Pico.  It is very possible to get started with Pico even with [no prior knowledge of HTML](#web-pages-without-html).  You'll find however that a little bit of coding knowledge will get you a long way.
+
+![NotePaper Theme for Pico]({{ site.github.url }}/style/images/docs/pico_about_notepaper_theme.jpg)
+NotePaper, a ready-to-go, community-developed theme.
+
+* Probably move this later, but I didn't want to forget to include it in the same commit as the image file.
+* Add example images for other themes.
+
+
+* insert some images showing Pico in use.  Borrow from readme?
+  * Can the site do thumbnails and lightboxing?
+* Add a style for these images? Their lack of a border or shadow makes them hard to distinguish from the page background.
+  * Again, can these be thumbnailed and lightboxed? That might work better.
+
+* Larger "About" portion at the beginning.
+  * Should contain all the "Why's" of Pico, before the "How's"
+
+Now let's dive into what makes Pico different from other solutions.
+
+## A Flat File CMS
+
+Pico is a Content Management System, or CMS.  If you've heard the term before, it's likely in the context of WordPress, Joomla, Drupal (intentionally not linking to the competition ;P) or many of the other popular CMS's on the block.  A Content Management System makes creating a beautiful and well-organized website easy and frustration free.  It can also help those without extensive knowledge of programming on the web to achieve the same professional quality website as those that do.
+
+ * Last sentence is a bit rough....
+ * Add headers for CMS and Flat file or is that excessive?
```

Comment:
Do you think I should trim down the "What is a CMS" type content then?

---

As far as "first offenders" go, I have an interesting (if off-topic) example.

My (long-time) girlfriend wanted to set up a personal site for some writing she's done on various other websites.  I described both Pico and WordPress, and she wanted to try WordPress for its familiarity and ease-of-use.  She hadn't used it before, but it was familiar just in the sense that the workflow is the same as submitting things to other sites.

As far as computers go, I'd call her somewhere between intermediate and advanced (she knows a lot more than she gives herself credit for, but often lacks the confidence to act on that knowledge).

After I set it up for her, she spends about 30 minutes going through all the options, trying to personalize it and make it work *just right*.  She then goes "yeah, this is too complicated.  I don't need all this stuff, I just want to post my writing".

Now she's using Pico and NotePaper. :laughing: 

---

So personally, I'd love Pico to be the first option someone tries.  That'll probably never be the case though.